raiden/ui/cli.py

```diff
@@ -493,3 +497,26 @@ def append_report(subject, data):
     else:
         print('[5/5] smoketest had errors, report was written to {}'.format(report_file))
         sys.exit(1)
+
+
+@run.command()
+@OPTION_ADDRESS
+@OPTION_DATADIR
+def removedb(address, datadir):
```

Comment:
[Here is the relevant part](http://click.pocoo.org/6/commands/#nested-handling-and-contexts) of the click docs about how to pass values between (sub-)commands. Also konrad already uses this to some extent for the smoketest.